pkg/security/security_profile/tests/activity_tree_test.go

```diff
@@ -163,7 +163,7 @@ func TestActivityTree_CreateProcessNode(t *testing.T) {
 			differentContainerID:  true,
 			resultNodeShouldBeNil: true,
 			resultNewProcessNode:  false,
-			resultErr:             activity_tree.ErrContainerIDNotEqual,
+			resultErr:             nil,
```

Comment:
Hey 👋🏻 
Yes, the container ID error isn't really an error when you think about it: it will happen every time you try to insert an event. I think we can keep the test still as it is a good way to make sure we do check the activity tree selector: success is detected if the returned tree is empty. What do you think ?

Reply:
I don't have strong opinion on that, both works for me :+1: